packages/components/src/components/Flag/index.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import React from 'react';
+import { FLAGS } from './flags';
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+`;
+
+const Img = styled.img``;
+
+interface Props {
+    className?: string;
+    country: keyof typeof FLAGS;
+    size?: number;
+    onClick?: () => void;
+}
+
+const Flag = ({ size = 24, country, className, onClick }: Props) => {
+    return (
+        <Wrapper>
+            <Img
+                // eslint-disable-next-line import/no-dynamic-require, global-require
+                src={require(`../../images/flags/${country.toLowerCase()}.svg`)}
```

Comment:
Not a big fan of dynamic requires in the markup, it feels error prone but then I'm also not sure how exactly the build is set up for this in Next. 

For me, if the SVGs are being output to a path that can be reached, we could just set the path correctly (with the right asset prefix) or we could dynamically load the SVG and inject it instead of using an image tag (we could maybe even use React.lazy and Suspend with that).

Reply:
Injecting SVGs is not really good for performance. 
I already tried it and it's too slow. Maybe this solution is a bit better? no eslint or ts ignore :) 

https://github.com/trezor/trezor-suite/pull/2988/files#diff-6d4e41c97f9a02ff96e8fecf6ddf0834b1beda7fff118e03f1022704b8814de2R21